src/test/java/com/clova/anifriends/domain/auth/jwt/JJwtProviderTest.java

```diff
@@ -0,0 +1,165 @@
+                () -> jJwtProvider.parseAccessToken(invalidAccessToken))
+                .isInstanceOf(InvalidJwtException.class);
+        }
+
+        @Test
+        @DisplayName("예외(ExpiredAccessTokenException): 만료된 토큰")
+        void exceptionWhenExpiredJwt() {
+            //given
+            int expirySeconds = -1;
+            JJwtProvider expiredJJwtProvider = new JJwtProvider(issuer, expirySeconds,
+                refreshExpirySeconds, testSecret, testRefreshSecret);
+            UserToken userToken = expiredJJwtProvider.createToken(userId, roleVolunteer);
+            String expiredAccessToken = userToken.accessToken();
+
+            //when
+            //then
+            assertThatThrownBy(() -> jJwtProvider.parseAccessToken(expiredAccessToken))
+                .isInstanceOf(ExpiredAccessTokenException.class);
+        }
+
+        @Test
+        @DisplayName("예외(InvalidJwtException): 리프레시 토큰 사용 불가")
+        void exceptionWhenUsingRefreshToken() {
+            //given
+            String refreshToken = userToken.refreshToken();
+
+            //when
+            //then
+            assertThatThrownBy(() -> jJwtProvider.parseAccessToken(refreshToken))
+                .isInstanceOf(InvalidJwtException.class);
+        }
+    }
+
+    @Nested
+    @DisplayName("RefreshAccessToken 메서드 실행 시")
+    class RefreshAccessTokenTest {
+
+        @Test
+        @DisplayName("성공")
+        void refreshAccessToken() {
+            //given
+            String refreshToken = userToken.refreshToken();
+
+            //when
+            //then
+            await().atLeast(10, TimeUnit.MILLISECONDS).untilAsserted(() ->
+            {
+                UserToken newUserToken = jJwtProvider.refreshAccessToken(refreshToken);
+                assertThat(newUserToken.accessToken()).isNotEqualTo(refreshToken);
+            });
+        }
+
+        @Test
+        @DisplayName("예외(InvalidJwtException): 액세스 토큰 사용 불가")
+        void exceptionWhenUsingAccessToken() {
+            //given
+            String accessToken = userToken.accessToken();
+
+            //when
+            //then
+            assertThatThrownBy(() -> jJwtProvider.refreshAccessToken(accessToken))
+                .isInstanceOf(InvalidJwtException.class);
+        }
+
+        @Test
+        @DisplayName("예외(ExpiredRefreshTokenException): 만료된 리프레시 토큰")
+        void exceptionWhenRefreshTokenIsExpired() {
+            //given
+            int refreshExpirySeconds = -1;
+            JJwtProvider expiredJJwtProvider = new JJwtProvider(issuer, expirySeconds,
+                refreshExpirySeconds, testSecret, testRefreshSecret);
+            UserToken userToken = expiredJJwtProvider.createToken(userId, roleVolunteer);
+            String expiredRefreshToken = userToken.refreshToken();
+
+            //when
+            //then
+            assertThatThrownBy(() -> jJwtProvider.refreshAccessToken(expiredRefreshToken))
+                .isInstanceOf(ExpiredRefreshTokenException.class);
```

Comment:
```suggestion
            // when
            Exception exception = catchException(
                () -> jJwtProvider.refreshAccessToken(expiredRefreshToken)));

            // then
            assertThat(exception).isInstanceOf(ExpiredRefreshTokenException.class);
```

로 excpetion을 then절에서 assert 하면 given/then 절이 더 명확해질것같습니다!